### ISSUE TITLE
State Machine

### ISSUE THREAD

**Issue by dronov** [samuels410/helpalife#6]
Hi. I think I found a better solution for updating state of blog posts. 
No need to keep actions 'publish, undo_publish' in controllers like [here] (https://github.com/samuels410/helpalife/blob/master/app/controllers/blog/posts_controller.rb#L63). There is a better solution to add [State Machine] (https://github.com/pluginaweek/state_machine) to Post model for keeping the right business logic. In my opinion controllers should'not make actions like these and the Post model will be with more clearer code.

If you're agreed I can fix it later after #5

**Comment by samuels410** [comment]
@dronov @lucasocon I am thinking we can Install wordpress to provide the blogging solution.
Sorry @lucasocon  when we develop the blog feature  I haven't think about that. This week I will Install the WP and map blog.helpalife.in domain.

**Comment by lucasocon** [comment]
Okay @samuels410 no problem.

**Comment by dronov** [comment]
@samuels410 :+1: 
Could I make new PR with removing blog post models/controllers and link_to blog.helpalife.in?

**Comment by samuels410** [comment]
@dronov Please do that.

**Comment by dronov** [comment]
@samuels410 ok, I will make it first. Please comment another issue #5

**Comment by samuels410** [comment]
Blog post moved to wordpress.
